### DISCUSSION TITLE
Drop support for AIX?

### DISCUSSION THREAD

**Discussion by jmwilson** [scsitape/stenc#57] [General]
None of the maintainers have access to an AIX build environment and there doesn't appear to be a turnkey low-cost way to get a hosted VM. Without knowing even the available compiler, there's no way to plan around the language standard to use, and whether the code even compiles.

**Comment by sunwire** [comment]
I am in favor of this idea.

**Comment by jonasstein** [comment]
I agree. If we have a developer with AIX someday in the future, we can still merge a PR to enable AIX support. @ninthclowd  any objections to dropping AIX?

**Comment by c3rberus** [comment]
It would be great if we can keep AIX support, let me see if I can put together a build system for this, I have a system available (Power8 AIX 7.1), it does not have the IBM XL C/C++ compiler, but I should be able to use the packages from AIX Toolbox to build.

**Reply by jonasstein** [reply]
Thank you. In case that you have a reliable source for AIX, we can add it again. 
When do you expect that your system is running? 
Please add yourself to https://github.com/scsitape/stenc/issues/36 as tester and prepare a pull request for a fully working and tested AIX source.

**Reply by jonasstein** [reply]
@c3rberus any news? Could you test it? Please add yourself to https://github.com/scsitape/stenc/issues/36 as tester and prepare a pull request for a fully working and tested AIX source.
